enterprise/server/remote_execution/operation/operation.go

```diff
@@ -59,6 +61,19 @@ type FinishWithErrorFunc func(finalErr error) error
 
 func GetStateChangeFunc(stream StreamLike, taskID string, adInstanceDigest *digest.ResourceName) StateChangeFunc {
 	return func(stage repb.ExecutionStage_Value, execResponse *repb.ExecuteResponse) error {
+		if stage == repb.ExecutionStage_COMPLETED {
+			if target, err := flagutil.GetDereferencedValue[string]("executor.app_target"); err == nil {
+				if u, err := url.Parse(target); err == nil && u.Hostname() == "cloud.buildbuddy.io" {
+					if execResponse.GetMessage() != "" {
+						execResponse.Message = execResponse.GetMessage() + "\n"
+					}
+					execResponse.Message = (execResponse.GetMessage() +
+						"The build used the old BuildBuddy endpoint, cloud.buildbuddy.io. " +
```

Comment:
nit, when I ran a build locally, the log message sort of blends in, because Bazel doesn't prefix it with "message from executor" or anything like that:

<img width="636" alt="image" src="https://user-images.githubusercontent.com/2414826/178777097-7cfe6a6e-4ae5-45ea-83d1-3105550bcf7c.png">


 Maybe we can add our own prefix? like:

```suggestion
						"NOTE: this BuildBuddy executor is configured to use the old BuildBuddy endpoint, cloud.buildbuddy.io. " +
```